### ISSUE TITLE
Integrate Jenkins on EIT's Slack team 

### ISSUE THREAD

**Issue by raquelalegre** [EIT-team/ScouseTom#25]
It'd be nice to have this integration done for the EIT jobs that exist on Jenkins at the moment (EIT*).
I have tried, but found a small problem and I don't have time to look into it, so I'm passing this to Sinan by now, although this has very low priority.

Some instructions that can be useful:
1. From the EIT-team channel, select **Configure Integrations**, add **Jenkins CI** and select the **#eitforthewin** channel. 
2. The **Jenkins plugin for Slack** has already been installed in production Jenkins by someone from RITS, so you can go directly to http://development.rc.ucl.ac.uk/jenkins, then select **Manage Jenkins** > **Configure System**  and see **Global Slack Notifier Settings**.
3. The plugin is already configured for the RITS slack team, so we need to find a way to **add a second team** there, since [they say this is possible since version 1.7 of the plugin](https://github.com/jenkinsci/slack-plugin/issues/10). Note we have version 1.8, check by going to **Manage Jenkins** > **Manage Plugins** > **Installed** > **Slack Notification Plugin**.  
4. Once this is done, all the EIT jobs can be configured to print out Jenkins notifications on Slack. Not sure how to put that on the configuration yaml files, but if you go to [the configuration of any EIT job, like this one](http://development.rc.ucl.ac.uk/jenkins/view/EIT/job/EITScouseTom-main/configure), you'll see a **Slack Notifications** section where you can select from this list of options:
   - Notify Build Start  
   - Notify Aborted  
   - Notify Failure  
   - Notify Not Built  
   - Notify Success  
   - Notify Unstable  
   - Notify Back To Normal
